geode-protobuf/src/main/proto/clientProtocol.proto

```diff
@@ -66,7 +66,7 @@ message Response {
         RemoveAllResponse removeAllResponse = 7;
         ListKeysResponse listKeysResponse = 8;
 
-        ErrorResponse errorResponse = 13;
+        Error error = 13;
```

Comment:
The reasoning is, we're not just using it as an ErrorResponse at the same level as a PutResponse or GetResponse -- it gets embedded in other responses now.